src/operations/aggregate.ts

```diff
@@ -47,14 +42,11 @@ export class AggregateOperation<T = Document> extends CommandOperation<T> {
   pipeline: Document[];
   hasWriteStage: boolean;
 
-  constructor(parent: OperationParent, pipeline: Document[], options?: AggregateOptions) {
-    super(parent, options);
+  constructor(pipeline: Document[], ns?: MongoDBNamespace, options?: AggregateOptions) {
+    super(undefined, Object.assign({ dbName: ns?.db }, options));
```

Comment:
```suggestion
    super(undefined, { dbName: ns?.db, ...options });
```

I'm surprised the super class permits the first argument to be optional but if that's the case my only comment here is to use some modern syntax for the same thing.